musonant/questioner
# Implement Get all meetups

#### What does this PR do?
- Implement Get all meetups
#### Description of Task to be completed?
- Enforce JS Doc for ESLint
- Write tests for meetup endpoints
- Setup Model
- Setup Meetup Controller
#### How should this be manually tested?
- pull branch
- _npm install_ and _npm run start_
- using postman test GET http://localhost:8000/api/v1/meetups. This should return an array of existing meetups with status code of 200.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162856693
#### Screenshots (if appropriate)
None
#### Questions:`
None
